src/System.Globalization/tests/CharUnicodeInfo/CharUnicodeInfoTestData.cs

```diff
@@ -0,0 +1,224 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+
+namespace System.Globalization.Tests
+{
+    public static class CharUnicodeInfoTestData
+    {
+        private static List<CharUnicodeInfoTestCase> s_testCases;
+        public static List<CharUnicodeInfoTestCase> TestCases
+        {
+            get
+            {
+                if (s_testCases == null)
+                {
+                    s_testCases = new List<CharUnicodeInfoTestCase>(); 
+                    string fileName = "UnicodeData.8.0.txt";
```

Comment:
Yes I asked or adding the version number to the file name to be easy identify the version of the Unicode we currently support.